pyrallis/config.py

```diff
@@ -0,0 +1,22 @@
+from enum import Enum
+from typing import Union
+
+
+class ConfigType(str, Enum):
+    YAML = 'yaml'
+    JSON = 'json'
+    TOML = 'toml'
+
+
+class Config:
```

Comment:
@idow09, any thoughts about the `config.py` file? My concern is that the naming here becomes kind of confusing, as `config.py` relates to general global config values and `config.Config` is just one of possibly many global config classes